src/NVD.hs

```diff
@@ -113,6 +113,24 @@ rebuildDB = do
         , "  published text,"
         , "  modified text)"
         ]
+    execute_ conn $
+      Query $
+      T.unlines
+        [ "CREATE TABLE cpes ("
+        , "  cve_id text REFERENCES cve,"
+        , "  part text,"
+        , "  vendor text,"
+        , "  product text,"
+        , "  version text,"
+        , "  \"update\" text,"
+        , "  edition text,"
+        , "  language text,"
+        , "  software_edition text,"
+        , "  target_software text,"
+        , "  target_hardware text,"
+        , "  other text,"
+        , "  UNIQUE(cve_id, part, product, version, \"update\",edition, language, software_edition, target_software, target_hardware, other))"
```

Comment:
The reason I added a bunch of deduplication is because with the previous version of the feeds, there were two overlapping definitions of vulnerable versions, so many were added twice. This was resolved with feed version 1.1. For the matchers/cpes table, the `DELETE FROM `followed by an `INSERT INTO` should be fine, a uniqueness constraint like this will only slow it down, it should no longer be needed. Even if duplicates occasionally end up in the database, it's not an issue if a cve matches twice.